### PR TITLE
PMM-7890 IA common tests

### PR DESCRIPTION
**Pull request by yurkovychv**
tests are [passing](https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/7893/tests)

### CODE REVIEW

File: pmm-app/tests/ia/pages/iaCommonPage.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    alertRules: 'Alert Rules',
+    ruleTemplates: 'Alert Rule Templates',
+    notificationChannels: 'Notification Channels',
+  },
   elements: {
     noData: locate('$table-no-data').find('h1'),
     pagination: '$pagination',
     itemsShown: '$pagination-items-inverval',
     rowInTable: locate('$table-tbody').find('tr'),
+    tab: (tabName) => `//li[@aria-label="Tab ${tabName}"]`,
```
Comment (puneet0191) [inline pmm-app/tests/ia/pages/iaCommonPage.js:20]: locate?

File: pmm-app/codeceptConfigHelper.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   pageObjects: {
     I: './tests/custom_steps.js',
+    codeceptjsConfig: './pr.codecept.js',
```
Comment (puneet0191) [inline pmm-app/codeceptConfigHelper.js:4]: what is this for?

File: pmm-app/tests/ia/pages/iaCommonPage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/iaCommonPage.js:20]: will try

File: pmm-app/codeceptConfigHelper.js (hunk 2 above)
Comment (puneet0191) [reply pmm-app/codeceptConfigHelper.js:4]: okay I see

**Review by puneet0191 — APPROVED** [review]

File: pmm-app/codeceptConfigHelper.js (hunk 2 above)
Comment (yurkovychv) [reply pmm-app/codeceptConfigHelper.js:4]: to simplify using it. like [here](https://github.com/percona/grafana-dashboards/blob/8b4b2d80b5c9d27ed104c9888079771666dae201/pmm-app/tests/pages/api/settingsAPI.js#L5)

File: pmm-app/tests/ia/pages/iaCommonPage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/iaCommonPage.js]: done, thanks

**Review by puneet0191 — DISMISSED** [review]

File: pmm-app/tests/pages/pmmSettingsPage.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const { I, adminPage } = inject();
 const assert = require('assert');
 
 const locateLabel = (dataQA) => locate(`[data-qa="${dataQA}"]`).find('span');
+const communicationDocsLink = 'https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/configure.html#communication';
```
Comment (atymchuk) [inline pmm-app/tests/pages/pmmSettingsPage.js:5]: nit: I would suggest we put all URLs into one object (one place) and reference it wherever needed. So much easier to reason about them all :)

**Review by atymchuk — APPROVED** [review]

File: pmm-app/tests/pages/pmmSettingsPage.js (hunk 3 above)
Comment (atymchuk) [reply pmm-app/tests/pages/pmmSettingsPage.js:5]: Right now they are spread all over the code.

Comment (yurkovychv) [reply pmm-app/tests/pages/pmmSettingsPage.js:5]: makes sense, will do, thanks

**Review by atymchuk — APPROVED** [review]
woohoo, great!

**Pull request merged**